.env.template

```diff
@@ -1,6 +1,16 @@
+RAISE_ON_MISSING_TRANSLATIONS=false
```

Comment:
I'd vote for this to be true by default - we hit a number of issue in TV when missing translation were causing havoc in production and were missed completely in test and development.

Reply:
I'd also vote for that, however I didn't want to turn it on by default as that would be a behavioural change. Which is why it falls back to false in the actual config file.

However, considering this is just the example .env file and it'd only turn it on for people newly setting up the project, I think now would be a good time to make that change.

@BroiSatse @shjohnson What do you think about:
a) Setting this to true in .env.example, so people setting up the project from scratch have it on by default? This is backwards compatible with existing setups.
and b) Setting this to true in development.rb so you have to turn it off rather than turning it on? This is not backwards compatible and would change this for anyone with an existing setup.